Docs/gtemu.c

```diff
@@ -106,7 +106,7 @@ int main(int argc, char *argv[])
     int hSync = (T.OUT & 0x40) - (S.OUT & 0x40); // "VGA monitor" (use simple stdout)
     int vSync = (T.OUT & 0x80) - (S.OUT & 0x80);
     if (vSync < 0) vgaY = -36; // Falling vSync edge
-    if (vgaX++ < 200) {
+    if (vgaX++ <= 200) {
       if (hSync) putchar('|');              // Visual indicator of hSync
       else if (vgaX == 200) putchar('>');   // Too many pixels
```

Comment:
Maybe Marcel can comment, but I wonder if the vgaX == 200 check should be first before the hSync check?

Reply:
The intent of the code is the following:

1. Scanlines emit characters up to and including 200 cycles (0..199)
2. Scanlines _longer_ than 200 cycles get truncated with '>'
3. Scanlines _different_ from 200 cycles get marked with '~'

Indeed, vgaX won't count beyond 200, but that's not needed for the above. It _does_ reach 200 however. vgaX++ is post-increment.

Reply:
Now I understand you. The original code is correct.